tests/testthat/test-monte-carlo-sampling.R

```diff
@@ -43,12 +43,12 @@ test_that("Analytical Solution is close to known solution", {
   )
   
   expect_equal(
-    analyticalSolution$geoMean,
+    analyticalSolution$`geo mean`,
```

Comment:
This was the test implemented to check that the Monte Carlo gave appropriate solutions:

1) Use `rlnorm()` function to simulate 2 virtual population with **known** log normal distributions
   a) Reference population: $mean log=0$ and $sd log = 1$
   b) Compared population: $mean log=1$ and $sd log = 2$
2) Compute expected $geoMean$ and $geoSD$ of their ratio from the analytical formula
$geoMean = exp(mean log_{compared} - mean log_{reference}) = exp(1-0) = e$
$geoSD = exp(\sqrt{sd log_{compared}^2+sd log_{reference}^2}) = \frac{e^5}{2}$
3) Assess if analytical solution from simulated populations match expected
4) Assess if Monte Carlo solution from simulated populations match expected